resotocore/resotocore/db/db_access.py

```diff
@@ -86,30 +91,72 @@ async def start(self) -> None:
         await self.pending_deferred_edge_db.create_update_schema()
         await self.package_entity_db.create_update_schema()
         for graph in cast(List[Json], self.database.graphs()):
-            log.info(f'Found graph: {graph["name"]}')
-            db = self.get_graph_db(graph["name"])
+            graph_name = GraphName(graph["name"])
+            log.info(f"Found graph: {graph_name}")
+            db = self.get_graph_db(graph_name)
             await db.create_update_schema()
+            await self.get_graph_model_db(graph_name)
         await self.cleaner.start()
 
     async def stop(self) -> None:
         await self.cleaner.stop()
 
-    async def create_graph(self, name: GraphName) -> GraphDB:
+    def graph_model_name(self, graph_name: GraphName) -> str:
+        return f"{graph_name}_model"
```

Comment:
`_` is a valid character in graph names. So I could create the graph: `foo` and `foo_model` - which would break the system. Can we use a character that the user is not allowed to choose?